manifests/vrrp/instance.pp

```diff
@@ -205,6 +209,7 @@
   $track_script                                                           = undef,
   Optional[Array[String[1]]] $track_process                               = undef,
   Optional[Array[String[1]]] $track_file                                  = undef,
+  Optional[Array[String[1]]] $vrrp_track_file                             = undef,
```

Comment:
Is that an array of absolte paths? then you can use:
```suggestion
  Optional[Array[Stdlib::Absolutepath] $vrrp_track_file                             = undef,
```

Reply:
But then we should really use it for both types, no?